spec/models/working_group_spec.rb

```diff
@@ -15,8 +15,24 @@
 
       it "returns an empty array" do
         expect(working_group.headers).to eq([])
-        expect(content_item.headers).to eq([])
+      end
+    end
+  end
+
+  describe "#policies" do
+    context "when there are no headers in the details hash" do
```

Comment:
Is this context meant to say `"when there are headers in the details hash"`?

Reply:
Good spot - updated!